File: msgraph-mail/go/utilities/users/mail_folder.go
package users
import (
    "github.com/microsoft/kiota/abstractions/go/serialization"
)
type MailFolder struct {
    childFolderCount *int32;
    childFolders []MailFolder;
    displayName *string;
    isHidden *bool;
    messageRules []MessageRule;
    messages []Message;
    multiValueExtendedProperties []MultiValueLegacyExtendedProperty;
    parentFolderId *string;
    singleValueExtendedProperties []SingleValueLegacyExtendedProperty;
    totalItemCount *int32;
    unreadItemCount *int32;
}

File: msgraph-mail/swiftconsoleapp/Sources/swiftconsoleapp/Models/Microsoft/Graph/MailFolder.swift
extension Swiftconsoleapp.Models.Microsoft.Graph {
public class MailFolder : Entity {
public var childFolderCount: Int?
public var childFolders: [MailFolder]
public var displayName: String
public var isHidden: Bool?
public var messageRules: [MessageRule]
public var messages: [Message]
public var multiValueExtendedProperties: [MultiValueLegacyExtendedProperty]
public var parentFolderId: String
public var singleValueExtendedProperties: [SingleValueLegacyExtendedProperty]
public var totalItemCount: Int?
public var unreadItemCount: Int?
}
}
